pkg/jsonmessage/jsonmessage.go

```diff
@@ -189,13 +189,9 @@ func DisplayJSONMessagesStream(in io.Reader, out io.Writer, terminalFd uintptr,
 				if isTerminal {
 					fmt.Fprintf(out, "\n")
 				}
-			} else {
-				diff = len(ids) - line
 			}
-			if isTerminal {
-				// NOTE: this appears to be necessary even if
-				// diff == 0.
-				// <ESC>[{diff}A = move cursor up diff rows
+			diff = len(ids) - line
```

Comment:
I guess I missed a step or two but from what I know in line 179 if `jm.ID` is not in `id` then Go initializes `line` to 0: https://play.golang.org/p/CEz0r8eWSn